templates/e2e-dspy-workflow/configs/training/lora/llama-3-8b.yaml

```diff
@@ -1,6 +1,4 @@
-model_id: meta-llama/Meta-Llama-3-8B-Instruct # <-- change this to the model you want to fine-tune
-train_path: s3://llm-guide/data/viggo/train.jsonl # <-- change this to the path to your training data
-valid_path: s3://llm-guide/data/viggo/val.jsonl # <-- change this to the path to your validation data. This is optional
+model_id: meta-llama/Meta-Llama-3-8B-Instruct # <-- change this to the model you want to fine-tunetrain_path: s3://llm-guide/data/viggo/train.jsonl # <-- change this to the path to your training data
```

Comment:
is this intended? `train_path` should be in the next line?

Reply:
Both removed because there technically should be no default (valid_path wasn't getting set to None, so eval loss on WANDB was from viggo